h2o-py/tests/testdir_apis/H2O_Module/pyunit_h2ocluster_shutdown.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import print_function
+import sys
+sys.path.insert(1,"../../../")
+from tests import pyunit_utils
+import h2o
+import threading
+
+def h2ocluster_shutdown():
+    """
+    Python API test: h2o.cluster().shutdown(prompt=False)
+    """
+    thread = threading.Thread(target=call_shutdown)
+    thread.daemon =True
+
+    try:
+        thread.start()
+        thread.join(1.0)
```

Comment:
not sure why you want to shut down a cluster from a separate thread... 
on the other hand, this runs on a testing framework, with cluster pre-readied -- shutting it down (and not restarting afterwards) might have bad consequences